kafka_consumer/datadog_checks/kafka_consumer/client/kafka_python_client.py

```diff
@@ -241,41 +175,31 @@ def _create_kafka_admin_client(self, api_version):
         return kafka_admin_client
 
     def _create_kafka_client(self, clazz):
-        kafka_connect_str = self.check.instance.get('kafka_connect_str')
-        if not isinstance(kafka_connect_str, (string_types, list)):
+        if not isinstance(self.config._kafka_connect_str, (string_types, list)):
             raise ConfigurationError('kafka_connect_str should be string or list of strings')
-        kafka_version = self.check.instance.get('kafka_client_api_version')
-        if isinstance(kafka_version, str):
-            kafka_version = tuple(map(int, kafka_version.split(".")))
-
-        tls_context = self.check.get_tls_context()
```

Comment:
`tls_context` is now being passed rather than being generated inside this function now that we no longer have access to `self.check.get_tls_context()`.